packages/nodes-base/nodes/ExecuteWorkflow/ExecuteWorkflow/ExecuteWorkflow.node.ts

```diff
@@ -14,9 +14,30 @@ import { loadWorkflowInputMappings } from './methods/resourceMapping';
 import { generatePairedItemData } from '../../../utils/utilities';
 import { getWorkflowInputData } from '../GenericFunctions';
 
+function getWorkflowInputValues(this: IExecuteFunctions) {
+	const newItems: INodeExecutionData[] = [];
+	const inputData = this.getInputData();
+
+	for (const [itemIndex, item] of inputData.entries()) {
+		const itemFieldValues = this.getNodeParameter('workflowInputs.value', itemIndex, {}) as {};
```

Comment:
I would suggest avoiding `{}` as a type. We can use our own `IDataObject` for this.
```suggestion
		const itemFieldValues = this.getNodeParameter(
			'workflowInputs.value',
			itemIndex,
			{},
		) as IDataObject;
```

Reply:
yep